Assets/GothicVR/Scripts/UI/MoveSpeedController.cs

```diff
@@ -17,5 +18,6 @@ void ChangeMoveSpeed(float movespeed)
     {
         movecontroller.moveSpeed = movespeed;
         PlayerPrefs.SetFloat(ConstantsManager.I.moveSpeedPlayerPref, movespeed);
+        MovementTypeController.UpdateSpeedVariable(movespeed);
```

Comment:
if we use a constant for speed, then these two are not necessarily needed, at least the second one